dsgrid/dataset/dataset_schema_handler_one_table.py

```diff
@@ -110,21 +115,15 @@ def _check_one_table_data_consistency(self):
                     f"load_data records do not match dimension records for {name}"
                 )
 
-    def get_unique_dimension_rows(self):
-        """Get distinct combinations of remapped dimensions.
-        Check each col in combination for null value."""
+    def make_dimension_association_table(self) -> DataFrame:
         time_cols = set(self._get_time_dimension_columns())
         pivoted_cols = set(self._config.get_pivoted_dimension_columns())
         exclude = time_cols.union(pivoted_cols)
         if self._config.get_table_format_type() == TableFormatType.UNPIVOTED:
             exclude.add(VALUE_COLUMN)
         dim_cols = [x for x in self._load_data.columns if x not in exclude]
         df = self._load_data.select(*dim_cols).distinct()
-
-        dim_table = self._remap_dimension_columns(df, True).distinct()
-        check_null_value_in_unique_dimension_rows(dim_table, exclude_columns=exclude)
```

Comment:
@lixiliu In this file we were previously not explicitly checking for null values during dataset registration (now we are), but were checking for them during dataset submission (after mapping to project). I removed the latter check because, when we map stacked dimensions, we first filter out all cases of `to_id == NULL`. So then it should not be possible that we have null values, right? Please confirm. If you want to keep the check for safety, I can do so.

Reply:
Yeah, if we already check to make sure that no null value exists for each dimension in the dataset (other than id) during registration, I would not expect nulls to be there in the dimension combinations.